src/compiler/prioritization/procedure.md

```diff
@@ -73,7 +73,9 @@ We need all [`I-prioritize T-compiler`](https://github.com/rust-lang/rust/issues
 The procedure here follows the [General issues review process](#General-issues-review-process).
 
 Note: triagebot automatically creates a topic and notify @*WG-prioritization* members once an issue is labelled with `I-prioritize`
+
 Note #2: These lists should typically be empty when we are close to the meeting.
+
 Note #3: we should not have unprioritized regressions ([stable](https://github.com/rust-lang/rust/issues?q=is%3Aopen+label%3Aregression-from-stable-to-stable+-label%3AP-critical+-label%3AP-high+-label%3AP-medium+-label%3AP-low+-label%3AT-infra+-label%3AT-libs+-label%3AT-release+-label%3AT-rustdoc), [beta](https://github.com/rust-lang/rust/issues?q=is%3Aopen+label%3Aregression-from-stable-to-beta+-label%3AP-critical+-label%3AP-high+-label%3AP-medium+-label%3AP-low+-label%3AT-infra+-label%3AT-libs+-label%3AT-release+-label%3AT-rustdoc) and [nightly](https://github.com/rust-lang/rust/issues?q=is%3Aopen+label%3Aregression-from-stable-to-nightly+-label%3AP-critical+-label%3AP-high+-label%3AP-medium+-label%3AP-low+-label%3AT-infra+-label%3AT-libs+-label%3AT-release+-label%3AT-rustdoc)) and ideally regressions should have an assignee.
```

Comment:
It's displayed like:
![image](https://user-images.githubusercontent.com/25030997/88291690-9bd9f480-cd33-11ea-9e72-d1ce9cda3e13.png)

This is... less readability. Should have break lines imo.